test/test-build.html

```diff
@@ -17,14 +17,17 @@
 
     System.config({
       baseURL: './fixtures/test-tree/',
+      meta: {
+         'output/*': { format: 'register' }
+      },
       paths: {
         'traceur-runtime': '../node_modules/traceur/bin/traceur-runtime.js',
         'babel/external-helpers': '../node_modules/babel-core/external-helpers.js',
         'jquery-cdn': './jquery.js',
         'output/*': './output/*'
       },
       bundles: {
-        'output/tree-build.js': ['third.js', 'cjs.js', 'jquery.js', 'second.js', 'global.js', 'amd.js', 'first.js'],
+        'output/tree-build.js': ['first.js', 'second.js', 'third.js', 'cjs.js', 'jquery.js', 'global.js', 'amd.js'],
```

Comment:
Changing the dependency order in the build is a breaking change. We need to ensure that dependencies are reported in the same order.